providers/tokenbucket/limiter.go

```diff
@@ -0,0 +1,29 @@
+package tokenbucket
+
+// Implement Limiter interface.
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/juju/ratelimit"
+)
+
+// TockenBucketInterceptor implement tocken bucket algorithm.
+type TockenBucketInterceptor struct {
+	tokenBucket *ratelimit.Bucket
+}
+
+// Limit Implement Limiter interface
+func (r *TockenBucketInterceptor) Limit(_ context.Context) error {
+	// Take one token pro request. This call doesn't block.
```

Comment:
I think this was a mistake 😛 ?
```suggestion
	// Take one token per request. This call doesn't block.
```